### DISCUSSION TITLE
unable to connect to minio

### DISCUSSION THREAD

**Discussion by alekseyolg** [apache/airflow#29823] [Q&A]
### Apache Airflow version

2.5.1

### What happened

I add a connection like "Amazon Web Services"
```
{"endpoint_url": "http://myhost:9000", "aws_access_key_id": "myacces", "aws_secret_access_key": "mysecret"}
```
I press Test
An error
```
EndpointConnectionError' error occurred while testing connection: Could not connect to the endpoint URL: "https://sts.amazonaws.com/"
```

### What you think should happen instead

successful connection

### How to reproduce

Follow my steps.

### Operating System

Debian 11

### Versions of Apache Airflow Providers
```
apache-airflow-providers-amazon==7.1.0
apache-airflow-providers-celery==3.1.0
apache-airflow-providers-cncf-kubernetes==5.1.1
apache-airflow-providers-common-sql==1.3.3
apache-airflow-providers-docker==3.4.0
apache-airflow-providers-elasticsearch==4.3.3
apache-airflow-providers-ftp==3.3.0
apache-airflow-providers-google==8.8.0
apache-airflow-providers-grpc==3.1.0
apache-airflow-providers-hashicorp==3.2.0
apache-airflow-providers-http==4.1.1
apache-airflow-providers-imap==3.1.1
apache-airflow-providers-microsoft-azure==5.1.0
apache-airflow-providers-mysql==4.0.0
apache-airflow-providers-odbc==3.2.1
apache-airflow-providers-postgres==5.4.0
apache-airflow-providers-redis==3.1.0
apache-airflow-providers-sendgrid==3.1.0
apache-airflow-providers-sftp==4.2.1
apache-airflow-providers-slack==7.2.0
apache-airflow-providers-sqlite==3.3.1
apache-airflow-providers-ssh==3.4.0
```

### Deployment

Official Apache Airflow Helm Chart v 1.8.0

### Deployment details

Official chart

### Anything else

No.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by hussein-awala** [comment]
If you want to create the connection from the UI, you should pass the `endpoint_url` in Extras, as explained in the Minio [doc](https://blog.min.io/apache-airflow-minio/), and if you want to create it using python:
```python
Connection.from_json('{"conn_type": "aws", "login": "myaccess", "password": "mysecret", "extra":{"endpoint_url": "http://myhost:9000"}}')
``` 
and the same for CLI.

**Comment by Taragolis** [comment]
@alekseyolg Please have a look important section in  [Amazon Web Services Connection](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html#amazon-web-services-connection)
